cirq-core/cirq/protocols/json_test_data/PauliMeasurementGate.repr_inward

```diff
@@ -2,4 +2,3 @@
     cirq.PauliMeasurementGate((cirq.X, cirq.Y, cirq.Z), cirq.MeasurementKey(name='key')),
     cirq.PauliMeasurementGate((cirq.X, cirq.Y, cirq.Z), cirq.MeasurementKey(path=('p', 'q'), name='key')),
 ]
-
```

Comment:
nit: unrelated change.

Reply:
This was from my commit hook which balks at trailing whitespace.  Will put it back as it were.